identity-service/src/awsSNS.js

```diff
@@ -138,43 +141,77 @@ async function drainPublishedAnnouncements () {
 // we want it to continue
 async function drainMessageObject (bufferObj) {
   const { userId } = bufferObj
-  const { deviceType, awsARN, deviceToken } = bufferObj.device
   const { message, title, playSound } = bufferObj.notificationParams
 
-  try {
-    let formattedMessage = null
-    if (deviceType === 'ios') {
-      // TODO - add badge count here
-      formattedMessage = _formatIOSMessage(message, awsARN, playSound, title)
+  logger.debug(`Retrieving badge count for user ${userId}`)
+  const incrementBadgeQuery = await models.PushNotificationBadgeCounts.increment('iosBadgeCount', { where: { userId } })
+  // Parse the updated value returned from increment
+  // TODO: Is this actually the right access pattern?
+  const newBadgeCount = incrementBadgeQuery[0][0][0].iosBadgeCount
```

Comment:
There's probably a better way to do this. i don't know for sure but i'm willing to guess either `incrementBadgeQuery.iosBadgeCount` or `incrementBadgeQuery[0].iosBadgeCount` will work. The docs say the first return value is the model object with the properties https://sequelize.org/master/class/lib/model.js~Model.html#static-method-increment

Reply:
i don't think so...? The docs say its actually an array of models for each returned
`
Promise<Model[], ?number> | returns an array of affected rows and affected count 
`

Here is the actual output logged out:
```
[[[{\"userId\":3,\"iosBadgeCount\":3,\"createdAt\":\"2020-01-10T02:14:53.801Z\",\"updatedAt\":\"2020-01-13T18:21:07.028Z\"}],1]]
```